frontend/src/metabase/styled-components/containers/GlobalStyles/GlobalStyles.tsx

```diff
@@ -27,6 +27,13 @@ export const GlobalStyles = (): JSX.Element => {
       --mb-color-focus: ${color("focus")};
       --mb-color-bg-dark: ${color("bg-dark")};
       --mb-color-bg-light: ${color("bg-light")};
+
+      /*
+        Theming-specific CSS variables.
+        These CSS variables are not part of the core design system colors.
+      **/
+      --mb-color-bg-dashboard: var(--mb-color-bg-white);
+      --mb-color-bg-dashboard-card: var(--mb-color-bg-white);
```

Comment:
A self-reminder to also test if night mode is compatible, as night mode uses `bg-dark`